app/components/ProfileHeader/index.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+import PageSection from 'components/PageSection';
+import BorderBottomDiv from 'components/shared/BorderBottomDiv';
+
+// Easily keeps image/logo within good size range.
+const ProfileImg = styled.img`
+  min-height: 75px;
+  min-width: 75px;
+  max-height: 100px;
+  max-width: 100px;
+`
+
+const ProfileHeader = (props) => (
+  <div className="text-center">
+    <BorderBottomDiv>
+      <ProfileImg src={props.url} className="mt-4"
+        alt={props.guide ? 'guide logo' :'profile image'}
+      />
+      <h1 className="h2 text-info">{props.name}</h1>
+      {
+        // Only creates a location/summary element if one exists
+        (props.location || props.summary) &&
+        <h2 className="h5 mb-3 text-muted">{props.location ? props.location : props.summary}</h2>
+      }
+    </BorderBottomDiv>
+    <BorderBottomDiv>
+      <ul className="list-inline row m-2 justify-content-around">
+        {
+          Object.keys(props.stats).map((stat, i, arr) => {
+            return (
+              <li className="list-inline-item" key={i}>
+                <p className="h3 text-info">{props.stats[stat]}</p>
+                <p className="small text-muted mb-0">
+                  {/* Capitalize stat name and add space before camelcase or snakecase character (or number) */}
+                  { stat.slice(0, 1).toUpperCase() + stat.slice(1)
+                    .replace(/(_[a-z])|[_]?([A-Z0-9])/g, (match, p1, p2) => {
```

Comment:
This is way too complicated to render inline. If we needed this functionality I'd want it to be broken out into a separate function

Reply:
Instead of this, let's make the `stats` object be of a shape, e.g.

```
stats: PropTypes.objectOf(PropTypes.shape({
    count: PropTypes.number,
    label: PropTypes.string,
  })
```

In general, I don't want any complicated logic on the frontend. It will sometimes be needed, but encoding business logic here is a recipe for bugs down the line IMO

Reply:
OK. Wasn't sure what would be best to do here, so just went with my first thought and finished it to get it done and turn in (I also like working with RegEx).